api/src/pcapi/routes/public/documentation_constants/fields.py

```diff
@@ -98,6 +98,9 @@ class SomeOtheResponseModel(BaseModel):
     # Offer fields
     OFFER_ID = Field(description="Offer id", example=12345)
     OFFER_NAME = Field(description="Offer title", example="Le Petit Prince")
+    OFFER_DESCRIPTION = Field(
+        None, description="Offer description", example="A great book for kids and old kids.", max_length=1000
```

Comment:
Pas de `None` parce que ça ajoute une valeur par défaut (ce que l'on ne veut pas) 
```suggestion
        description="Offer description", example="A great book for kids and old kids.", max_length=1000
```